src/main/java/net/atos/zac/app/zaken/model/RESTZaakBetrokkeneGegevens.java

```diff
@@ -7,17 +7,24 @@
 
 import java.util.UUID;
 
+import jakarta.validation.constraints.NotNull;
+
 import net.atos.zac.app.klanten.model.klant.IdentificatieType;
 
 public class RESTZaakBetrokkeneGegevens {
 
+    @NotNull
     public UUID zaakUUID;
 
+    @NotNull
     public UUID roltypeUUID;
 
+    @NotNull
```

Comment:
If the value is not allowed to be empty (that's what I would normally expect with a non-null string), then we should use `jakarta.validation.constraints.NotBlank` validation.

Reply:
but there are exceptions because ZAC quite often uses empty strings I noticed. not sure if that applies here though. I would expect not